### PR TITLE
Create README.rst

### PR DESCRIPTION
**Pull request by WO15**
Content to display on https://www.odoo.com/apps/modules/8.0/connector_woocommerce/
Currently there is none.

### CODE REVIEW

File: connector_woocommerce/README.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
```
Comment (elicoidal) [inline connector_woocommerce/README.rst:5]: Hi,
Please follow: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Document title lines must be as long as the title

File: connector_woocommerce/README.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+{WooCommerce Connector}
```
Comment (elicoidal) [inline connector_woocommerce/README.rst:6]: remove {}

File: connector_woocommerce/README.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+==============
+
+
+This is an Open-Source connector linking Odoo and WooCommerce. It is capable to interface the following information
+* Import category
```
Comment (elicoidal) [inline connector_woocommerce/README.rst:11]: Can you please indicate who is master/slave and whether it is automated?

File: connector_woocommerce/README.rst [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  * sudo pip install woocommerce
+  * (Optional) check with command below if it went OK.
+      pip freeze | grep Woo
+
+
+Youtube video with extensive demonstration of the connector.
+https://www.youtube.com/watch?v=vz759wsZFak
+
+Wordpress module for Odoo WooCommerce Connector is available here.
+https://wordpress.org/plugins/connector-woo-odoo/
```
Comment (elicoidal) [inline connector_woocommerce/README.rst:35]: This readme is incomplete (please refer to official template

**Comment by parthivgls** [comment]
@elicoidal should I close this PR as we are planning to clean all pending PR for this repo.

**Comment by elicoidal** [comment]
pleaes go ahead @parthivgls

**Pull request closed without merging**